geomesa-ui/src/main/ui/src/app/unique/unique.js

```diff
@@ -0,0 +1,70 @@
+angular.module('geomesa.unique', [])
+
+    .controller('UniqueController', ['$scope', function($scope) {
+        var request = createWPSExecuteRequest('Who', 'true', 'ASC', 'true');
+        OpenLayers.Request.POST({
```

Comment:
Is it necessary to use the OpenLayers post function. Could we use $http or $resource instead? It would be nice to be as decoupled from openlayers as possible.